docs/modules/ROOT/pages/signers.adoc

```diff
@@ -57,11 +56,6 @@ The following table shows which signer types are compatible with each network ty
 |✅ Supported
 |❌ Not supported
 
-|`vault_cloud`
```

Comment:
Removed due to deprecation

https://developer.hashicorp.com/hcp/docs/vault-secrets/migrate-vault